skills/src/main/java/cm/aptoide/skills/SkillsFragment.kt

```diff
@@ -130,6 +127,37 @@ class SkillsFragment : Fragment(), PaymentView {
     }
   }
 
+
+  private fun setupPurchaseTicketLayout(
+          eSkillsPaymentData: EskillsPaymentData) {
+    setupQueueIdLayout()
+    if (eSkillsPaymentData.environment == EskillsPaymentData.MatchEnvironment.SANDBOX){
+      setupSandboxTicketButtons(eSkillsPaymentData)
+    }
+    else{
+      updateHeaderInfo(eSkillsPaymentData)
+      setupPurchaseTicketButtons(eSkillsPaymentData)
+    }
+    setupAppNameAndIcon(eSkillsPaymentData.packageName)
+  }
+
+  private fun setupSandboxTicketButtons(eSkillsPaymentData: EskillsPaymentData){
+    if (RootUtil.isDeviceRooted()) {
+      showRootError()
+    }
+    else {
```

Comment:
Please format the code properly according to our style guide
[AptoideSquareAndroid_v4.xml.txt](https://github.com/Catappult/appcoins-wallet-android/files/9506335/AptoideSquareAndroid_v4.xml.txt)